typings/index.d.ts

```diff
@@ -97,6 +97,28 @@ declare namespace CodeceptJS {
   interface Globals {
     codeceptjs: typeof codeceptjs;
   }
+
+  class Helper {
+    constructor(config: unknown)
+    static _config(): unknown
+    _validateConfig(config: unknown): unknown
+    _setConfig(opts: unknown): unknown
+    _init(): unknown
+    _before(): unknown
+    _after(): unknown
```

Comment:
These three in the test are expect to accept an argument but the base class implementation doesn't signify it.